plugins/secretstores/jwt/jwt.go

```diff
@@ -0,0 +1,141 @@
+package jwt
```

Comment:
Please add the `go:generate` statement to be able to embedd the `sample.conf` file into the readme.
```suggestion
//go:generate ../../../tools/readme_config_includer/generator
package jwt
```